src/main/resources/application.yaml

```diff
@@ -140,7 +142,8 @@ ftp:
       folder: FINREM
     - service: fpl_case_service
       folder: FPL
-
+    - service: prl_cos_api
+      folder: PRIVLAW
```

Comment:
Could you please add these lines in access-token config:
`
    - serviceName: prl_cos_api
      newContainerName: new-bulkprint
      validity: ${SAS_TOKEN_VALIDITY:300}`

don't worry about the container name. The storage containers are not in use at the moment but it doesn't work without this config.